x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/index.tsx

```diff
@@ -90,6 +111,8 @@ interface AllRulesProps {
  *   * Delete
  *   * Import/Export
  */
+// TODO: remove below eslint-disable
```

Comment:
Do not forget about that

Reply:
I think you can create a generic component for the table + tab to remove complexity form here